app/src/views/private/private-view.vue

```diff
@@ -146,7 +146,7 @@ const theme = computed(() => {
 
 provide('main-element', contentEl);
 
-router.afterEach(async (to, from) => {
+router.afterEach(async () => {
```

Comment:
Should we remove the `async` as well? Seems like an inner `await` logic was removed awhile ago in this commit: https://github.com/directus/directus/commit/6f9b2cafcdf8e2ec7ce7c0d52b9cce5a6fdff26e#diff-fd538d98b19483628862d186fd3a50605778a0e62df74f3eb01eaabddb6f2440 👍

LGTM other than that!

Reply:
Good idea 👍 Done!